helpers/test/test_hdatetime.py

```
@@ -671,5 +675,36 @@ def test4(self) -> None:
         with self.assertRaises(ValueError) as err:
             hdateti.str_to_timestamp(datetime_str, timezone_info)
         actual = str(err.exception)
-        expected = "Unknown string format: qwe28abc07-201234 present at position 0"
+        expected = (
+            "Unknown string format: qwe28abc07-201234 present at position 0"
+        )
+        self.assert_equal(actual, expected)
+
+
+# #############################################################################
+# Test__dassert_str_is_date
+# #############################################################################
+
+
+class Test__dassert_str_is_date(hunitest.TestCase):
+    def test_dassert_str_is_date_valid(self) -> None:
+        """
+        Test for valid date string(should pass).
+        """
+        date_str = "20221101"
+        hdateti._dassert_str_is_date(date_str)
+
+    def test_dassert_str_is_date_invalid(self) -> None:
+        """
+        Test for invalid date that has dashes(should raise an error).
```

. It should raise an error.